README.md

```diff
@@ -322,6 +322,7 @@ This repository also provides one such platforms where contributers come over an
 | [Tic-Tac-Toe Game](https://github.com/kunjgit/GameZone/tree/main/Games/Tic-Tac-Toe) |
 | [Rapid_click_frenzy](https://github.com/kunjgit/GameZone/tree/main/Games/Rapid_click_frenzy)                  
 
+| [Catch_The_Ball] (http://127.0.0.1:5501/Games/Catch_The_Ball/index.html)                              |
```

Comment:
Place name of the game in the end instead!!
eg catch_the_ball should be same the folder name

Reply:
@ishita-43 please check if it is alright now